templates/home.html

```diff
@@ -1,8 +1,11 @@
 {{ define "content" }}
 
 <h1 class="text-primary">HELLO GOLANG</h1>
-{{ range $flash := .flashes }}
-  <h2>{{ $flash }}</h2>
+{{ if .userID  }}
```

Comment:
It would better to have a presenter object for this to wrap view related variables together. But to keep things simple, having a better DSL with a better naming would help:

```suggestion
{{ if .authenticatedUser  }}
```

or the opposite:

```suggestion
{{ if .anomymousUser  }}
```

Reply:
Sounds better. Fixed in ef89d938577fe4c4c0af4346ea3496707cb17b55